src/main/java/cnLabs/unsplashDemo/Service/PexelService.java

```diff
@@ -0,0 +1,78 @@
+package cnLabs.unsplashDemo.Service;
+
+import cnLabs.unsplashDemo.Model.PexelsResponse;
+import cnLabs.unsplashDemo.Model.PhotoPexel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@Service
+public class PexelService {
+
+    @Autowired
+    @Qualifier("webClientPexels")
+    WebClient webClient;
```

Comment:
you could skip `@Qualifier` if it is ok to use value as a variable name. 

```suggestion
    WebClient webClientPexels;
```

Reply:
Thank you! I didn't  quite understand that.